### PR TITLE
fix allowE2Create convar

### PR DESCRIPTION
**Pull request by PLally**

### CODE REVIEW

**Comment by brandonsturgeon** [comment]
nice. I think I'm the one who originally wrote this. good to know it's been nonfunctional for > 1 year

File: lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua [hunk 1]
```diff
@@ -389,7 +389,7 @@ end
 
 __e2setcost(20)
 
-pewpew:CreateConVar("AllowE2Create", "int", true)
+pewpew:CreateConVar("AllowE2Create", "int", "1")
```
[the remark this answers is not in the document]
Comment (PLally) [reply lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua]: it just gets converted to a string if you use another type
https://github.com/CFC-Servers/PewPew/blob/master/lua/pewpewplugins/server/convars.lua#L29

[the remark this answers is not in the document]
Comment (brandonsturgeon) [reply lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua]: Does that mean you should treat it like a string later, or is it automatically converted to an int when you read it?

[the remark this answers is not in the document]
Comment (PLally) [reply lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua]: its automatically converted to an int

**Pull request merged**